hugegraph-core/src/main/java/org/apache/hugegraph/version/CoreVersion.java

```diff
@@ -41,6 +41,6 @@ public class CoreVersion {
     public static void check() {
         // Check version of hugegraph-common
         // TODO: why shall we check it? Update it if need
-        VersionUtil.check(CommonVersion.VERSION, "2.1.0", "2.2.0", CommonVersion.NAME);
+        VersionUtil.check(CommonVersion.VERSION, "1.0.0", "1.0.1", CommonVersion.NAME);
```

Comment:
maybe 1.0~1.2 first

Reply:
> 

i will fix it, but ApiVersion should discuss

Reply:
the right version is inclusive, `[1.0, 1.1)` is ok